### PR TITLE
Feature/data visualisation

### PR DESCRIPTION
**Pull request by ZenVega**
Hier eine Vorabversion der neuen Installationsdetails. Diese muss noch durch einige daten ergänzt werden, die erst nächste Woche reinkommen.

### CODE REVIEW

**Comment by ZenVega** [comment]
shadows still missing

**Review by uliSchuster — CHANGES_REQUESTED** [review]
Running the dashboard locally, I do see several errors in the console, mostly related to opening and closing the sidebar. the sidebar itself appears to be very wide now, covering most of the screen when open (tested using Chrome/Brave).

**Comment by uliSchuster** [comment]
The changes are live on staging for review.
One thing I noticed is that the chart does not include the date of the day shown (if the single-day tab is selected). Maybe you can add the day as additional item below the hour ticks?

**Comment by uliSchuster** [comment]
Even though the installation card says "Störung" on the overview, the card in the details view shows the last recorded sensor value and even claims that the air quality is good. The two cards should provide consistent information.

**Comment by ZenVega** [comment]
> 

Marco removed that "today" button in the visualization and so did I. I guess because the default state is the current time and it's also the last choosable option. Do you you think we should put it in again?: https://www.figma.com/file/MqtSmgOqoJHPuiMyqQ26Ae/Webseite?node-id=2476%3A30314

**Comment by uliSchuster** [comment]
Well, it's your decision of course. But as a user, I would find it odd to reload the page just to get back to today

File: src/views/Overview.vue [hunk 1]
```diff
@@ -62,7 +62,7 @@ const getLatestMeasurement = (sample) => {
       <div v-if="hasActiveSensors">
         <ul
           role="list"
-          class="grid grid-cols-1 gap-6 sm:grid-cols-2 xl:grid-cols-3"
+          class="lg:max-w-3xl flex w-full gap-6 overflow-scroll lg:flex-wrap"
```
[the remark this answers is not in the document]
Comment (ZenVega) [reply src/views/Overview.vue:65]: Hmm, I cannot reproduce this. What browser are you using? Did you try replacing it with overflow-auto and did it solve the problem?

[the remark this answers is not in the document]
Comment (rtzll) [reply src/views/Overview.vue:65]: Using Chrome, visiting https://staging.como-berlin.de/dashboard/1.
When I remove the scroll overflow it disappears. `overflow: auto` also removes it.

**Comment by ZenVega** [comment]
I fixed all comments except the one concerning the "today button". For that topic, I'd go for the UX-Designer's version. I'll however backcheck with him as soon as he's back and discuss this. Until then I'd say we leave it out. Could you push these changes to staging? Thanks

**Pull request merged**